### PR TITLE
fix(addon-mobile): `Sheet` fix initial scroll on iOS

### PR DESCRIPTION
**Pull request by waterplea**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

## What is the current behavior?

Closes #

### CODE REVIEW

File: projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts [hunk 1]
```diff
@@ -92,9 +92,21 @@ export class TuiSheetComponent<T> implements TuiSheetRequiredProps<T>, AfterView
...
+        const {nativeElement} = this.el;
+        const scrollTop = [...this.stops, this.sheetTop, this.contentTop][
             this.item.initial
         ];
+
+        // Sometimes on iOS it ignores initial scroll, so we try it 5 times over the course of 200ms
+        timer(0, 50)
+            .pipe(
+                take(5),
+                // eslint-disable-next-line rxjs/no-ignored-takewhile-value
```
Comment (waterplea) [inline projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:104]: Is it a dumb rule? Should we disable it?

File: projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts [hunk 2]
```diff
@@ -92,9 +92,21 @@ export class TuiSheetComponent<T> implements TuiSheetRequiredProps<T>, AfterView
     }
 
     ngAfterViewInit(): void {
-        this.el.nativeElement.scrollTop = [...this.stops, this.sheetTop, this.contentTop][
+        const {nativeElement} = this.el;
+        const scrollTop = [...this.stops, this.sheetTop, this.contentTop][
             this.item.initial
         ];
+
+        // Sometimes on iOS it ignores initial scroll, so we try it 5 times over the course of 200ms
```
Comment (nsbarsukov) [inline projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:100]: https://taiga-ui--pr4579-sheet-gpy4m57y.web.app/components/sheet#basic


https://github.com/Tinkoff/taiga-ui/assets/35179038/3d599fd2-6734-4dc3-999a-f24080c2cc5b


___
https://taiga-ui.dev/components/sheet#basic

https://github.com/Tinkoff/taiga-ui/assets/35179038/b03f3489-2c27-468a-b25f-5011d6add775

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts (hunk 2 above)
Comment (nsbarsukov) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:100]: iPhone 13 mini
ios 16.4.1

File: projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts (hunk 1 above)
Comment (splincode) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:104]: https://github.com/Tinkoff/taiga-ui/pull/4586

File: projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts (hunk 2 above)
Comment (waterplea) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:100]: Could you debug it by any chance? What are scrollTop values there?

File: projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts (hunk 1 above)
Comment (morenmoren123) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:104]: wondering if anyone could spare some time and really help me out. need to access a remote i no longer have.

File: projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts (hunk 2 above)
Comment (waterplea) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:100]: > 
> https://github.com/Tinkoff/taiga-ui/assets/35179038/5f97fcb5-b1f7-4bbc-8ab6-2fe8b1051967
> 
> 

Thank you. In your first video it sometimes shows properly, what numbers are there in that case?

Comment (nsbarsukov) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:100]: The same ones(

https://github.com/Tinkoff/taiga-ui/assets/35179038/6966d3cb-e157-450d-ba79-31cc8d51de81

Comment (nsbarsukov) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:100]: @waterplea I think that problem starts here

https://github.com/Tinkoff/taiga-ui/assets/35179038/1c6ee17d-bda2-4ba4-9c88-cf07f6c9d17d

Comment (waterplea) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:100]: I will try to look into it next week.

File: projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts (hunk 1 above)
Comment (morenmoren123) [reply projects/addon-mobile/components/sheet/components/sheet/sheet.component.ts:104]: really hoping someone can help me piece it together and how to deliver. thanks

**Pull request closed without merging**